cmd/types/types.go

```diff
@@ -6,3 +6,11 @@ type UserInfo struct {
 	Location    string `json:"location"`
 	PublicRepos int    `json:"public_repos"`
 }
+
+//PublicGist contains basic info of publi gist
+type PublicGist struct {
```

Comment:
Currently, the entire app is fetching only public data so please rename it into Gist

Reply:
And you might need gists list to store gist